src/components/Garden/Filters/ListFilter.js

```diff
@@ -35,10 +37,20 @@ function ListItem({ index, item, onSelect, selected }) {
           ? `
           color:${theme.content}
         `
-          : 'cursor:pointer'};
+          : 'cursor:default'};
       `}
     >
-      {item}
+      <span css="cursor:pointer" onClick={handleOnClick}>
+        {item}
+      </span>
+      <span
+        css={`
+          padding-left: ${1 * GU}px;
```

Comment:
Could we use margin instead? We generally use margin to give spacing between elements and padding for modifying the appearance of the element in question
```suggestion
          margin-left: ${1 * GU}px;
```